src/pages/Settings/Settings.tsx

```diff
@@ -273,9 +273,9 @@ const WebauthnRegistation = ({
 				open={needPrfRetry && !prfRetryAccepted}
 				onCancel={() => resolvePrfRetryPrompt(false)}
 			>
-				<h3 className="text-2xl mt-4 mb-2 font-bold text-primary dark:text-white">{t('pageSettings.registerPasskey.messageDone')}</h3>
-				<p className='dark:text-white'>{t('pageSettings.registerPasskey.passkeyCreated')}</p>
-				<p className='dark:text-white'>{t('pageSettings.registerPasskey.authOnceMore')}</p>
+				<h3 className="text-2xl mt-4 mb-2 font-bold text-primary dark:text-white">{t('registerPasskey.messageDone')}</h3>
+				<p className='dark:text-white'>{t('registerPasskey.passkeyCreated')}</p>
+				<p className='dark:text-white'>{t('registerPasskey.authOnceMoreSignup')}</p>
```

Comment:
Wait... on second thought, the message in `Settings.txt` probably shouldn't say "To finish setting up your wallet", because the wallet is already set up at that point. Rather it should say something like "To finish setting up your passkey", or just "To finish setting up" (which was probably why it was written like that in the first place).

So maybe we should keep `"authOnceMore": "To finish setting up, please authenticate with it once more.",` instead of `"authOnceMoreSignup": "To finish setting up your wallet, please authenticate with your passkey once more.",` after all?

Reply:
Yes you are right, so we keep the first